src/integrationtests/java/com/aws/greengrass/integrationtests/BaseITCase.java

```diff
@@ -148,4 +150,15 @@ public static void deleteWindowsTestUser(String username) throws IOException, In
         // Delete test user's home directory
         FileUtils.deleteQuietly(new File("C:\\Users\\" + username));
     }
+
+
+    @SuppressWarnings({"PMD.SignatureDeclareThrowsException"})
+    public static void overrideInstanceField(Object target, String field, Object value) throws Exception {
```

Comment:
what? Why does this exist

Reply:
oh, I didn't find a good way to capture FSS status updates in my integ tests to check rejection status. I had to override the deployment config with the spy version. I need this to replace the private deploymentConfig value.

Maybe you can suggest a better alternative, as my knowledge about FSS service setup is limited.

Reply:
This has been removed and we can circle back on this for listening to fss update messages for rejection status.